README.md

```diff
@@ -19,9 +19,16 @@ record using the Aerospike database.
 ```js
 const Aerospike = require('aerospike')
 
+// INSERT HOSTNAME AND PORT NUMBER OF AEROPSIKE SERVER NODE HERE!
 const config = {
-  hosts: '192.168.33.10:3000'
+  hosts: '192.168.33.10:3000',
```

Comment:
I tend to use the `AEROSPIKE_HOSTS` environment variable to set the server address(es) – I think we should at least point that option out as an alternative as well. That option is currently not very well documented, except in the [`Config#hosts` docs](https://docs.aerospike.com/apidocs/nodejs/config#hosts) and the [Getting Started tutorial](https://docs.aerospike.com/apidocs/nodejs/tutorial-getting_started.html).

Reply:
You are right. I will create a tutorial in a future release.  It is mentioned, but it could be emphasized.